packages/editor-ui/src/components/CredentialEdit/CredentialEdit.vue

```diff
@@ -256,7 +256,9 @@ export default mixins(showMessage, nodeHelpers).extend({
 
 		setTimeout(() => {
 			if (this.credentialId) {
-				if (!this.requiredPropertiesFilled) {
+				if (!this.requiredPropertiesFilled && this.credentialPermissions.isOwner === true) {
```

Comment:
where are the tests? 
![7VE](https://user-images.githubusercontent.com/4711238/234868418-0e1fca9a-23c1-44d2-bb5f-79abdf81ebe9.gif)

Reply:
thank you